src/components/ShareDrawModal.tsx

```diff
@@ -0,0 +1,47 @@
+import { Anchor, Modal, Stack, Text } from '@mantine/core';
+import Image from 'next/image';
+import Link from 'next/link';
+
+type Props = {
+  open: boolean;
+  onClose: () => void;
+};
+
+const ShareDrawModal = ({ open, onClose }: Props) => {
+  const drawUrl = 'devstaff.gr/draw';
+
+  return (
+    <>
+      <Modal
+        opened={open}
+        onClose={onClose}
+        centered
+        title="Participate to the draw"
```

Comment:
I think this is the correct grammar here.

```suggestion
        title="Participate in the draw"
```

Reply:
Seems that duolingo does not work :P.